doc/source/user_guide/io.rst

```diff
@@ -349,10 +349,38 @@ error_bad_lines : boolean, default ``True``
   returned. If ``False``, then these "bad lines" will dropped from the
   ``DataFrame`` that is returned. See :ref:`bad lines <io.bad_lines>`
   below.
+
+  .. deprecated:: 1.3
+     The ``on_bad_lines`` parameter takes precedence over this parameter
+     when specified and should be used instead to specify behavior upon
+     encountering a bad line instead.
 warn_bad_lines : boolean, default ``True``
   If error_bad_lines is ``False``, and warn_bad_lines is ``True``, a warning for
   each "bad line" will be output.
 
+  .. deprecated:: 1.3
+     The ``on_bad_lines`` parameter takes precedence over this parameter
+     when specified and should be used instead to specify behavior upon
+     encountering a bad line instead.
+on_bad_lines : {{None, 'error', 'warn', 'skip'}}, default ``None``
+    Specifies what to do upon encountering a bad line (a line with too many fields).
+    Allowed values are :
+
+        - ``None``, default option, defers to ``error_bad_lines`` and ``warn_bad_lines``.
+
+          Note: This option is only present for backwards-compatibility reasons and will
+          be removed after the removal of ``error_bad_lines`` and ``warn_bad_lines``.
+          Please do not specify it explicitly.
+
+        - 'error', raise an Exception when a bad line is encountered.
+        - 'warn', raise a warning when a bad line is encountered and skip that line.
```

Comment:
show a ParserWarning (I think)?

Reply:
IDK about this one. It seemed to just print stuff for me and not raise warnings?

Reply:
NVM. Confirmed that it prints a warning and changed message.